src/applications/dependents/view-dependents/tests/components/ViewDependentsListV2.unit.spec.jsx

```diff
@@ -85,4 +120,21 @@ describe('<ViewDependentsList />', () => {
       $$('va-button[text="Remove this dependent"]', container).length,
     ).to.equal(2);
   });
+
```

Comment:
Can you add a test case for when the date of birth object is not valid?

Also, do we need a test case for when the dependent is over 18? Mostly to confirm that a student who is over 18 doesn't see this message